lib/Sema/CMakeLists.txt

```diff
@@ -93,7 +93,7 @@ if (SWIFT_SWIFT_PARSER)
     SWIFT_SWIFT_PARSER
   )
   target_link_libraries(swiftSema PRIVATE
-    $<TARGET_OBJECTS:swiftASTGen>)
+    swiftASTGen)
```

Comment:
Linking to `swiftASTGen` target defaults the target link language `Swift` which caused issues.
This PR resolved it by explicitly setting the linker language.
`set_target_properties(${name} PROPERTIES LINKER_LANGUAGE CXX)`

Reply:
Nice. With that fixed I think I can put a PR up to do use FetchContent instead. Then no need for earlyswiftsyntax at all.